grails-app/controllers/org/pih/warehouse/core/LocationController.groovy

```diff
@@ -112,7 +112,11 @@ class LocationController {
                     if (locationInstance?.id == session?.warehouse?.id) {
                         session.warehouse = locationInstance
                     }
-                    flash.message = "${warehouse.message(code: 'default.updated.message', args: [warehouse.message(code: 'location.label', default: 'Location'), locationInstance.id])}"
+                    flash.message = []
+                    flash.message.add([code: 'default.updated.message', args: [warehouse.message(code: 'location.label', default: 'Location'), locationInstance.id]])
```

Comment:
Can it be done without initializing empty array? I mean merging those two lines into one:
```
flash.message = [[code: 'default.updated.message', args: [warehouse.message(code: 'location.label', default: 'Location'), locationInstance.id]]]
```